readthedocs_theme/templates/includes/footer.html

```diff
@@ -34,9 +34,8 @@ <h4 class="ui sub header">Learn more</h4>
           <div class="ui vertical inverted text menu">
             {% block menu_services %}
               <h4 class="ui sub header">Services</h4>
-              <a class="item" href="/pricing.html">Pricing</a>
-              <a class="item" href="/product.html">Features</a>
-              <a class="item" href="https://readthedocs.org">Open-source hosting</a>
```

Comment:
Seemed like this was odd, since we link to it in all our login/get started modals.